src/Experimental.cmake

```diff
@@ -106,6 +106,10 @@ set( EXPERIMENTAL_OPTIONS_LIST
    # PRL disabled 31 Aug 2021 because, at least on Mac, it misfires the
    # preference dialog whenever any keystroke shortcuts with Shift+ are used
    # EASY_CHANGE_KEY_BINDINGS
+
+   # DV 26 December 2023
+   # Use lib-wave-track-paint for in-memory waveform image generation
```

Comment:
I just merged a PR commit-experimental-branches and proposed another, maintain-experimental-branches.  I want to eliminate Experimental.cmake.

See the second PR.  If you are not certain to commit to using the experiment, instead define a bool constant in namespace Experimental.  Use if constexpr not ifdef to control new code.  That makes the compiler still check that it can build, when it does not build.

Reply:
If this PR passes QA, I think the proper solution would be to remove the old code altogether.